### DISCUSSION TITLE
weekly/2024-04-14

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#203] [Announcements]
# weekly/2024-04-14

① Bun 1.1 发布 - Bundows ② Signals 的标准化提案 ③ 免费的AI咒语聚合网站 ④ 前端框架新势力大盘点 ⑤ dashpress - 数据库管理 ⑥ node.js 截图库 ⑦ autospec - API 规范描述文件生成 ⑧ Journey.js - 交互式引导 ⑨ barbajs - 页面切换的平滑过渡 ⑩ CSS 容器查询交互式教程

https://sugarat.top/weekly/2024-04-14.html

**Comment by littlemoji** [comment]
质量真好！
> 弄个API文档生成最佳实践。

期待！
